Scripts/NuGet/EdFi.Installer.SandboxAdmin/Install-EdFiOdsSandboxAdmin.psm1

```diff
@@ -247,24 +242,17 @@ function Invoke-TransformWebConfigAccountInitialization {
         $Config
     )
     Invoke-Task -Name ($MyInvocation.MyCommand.Name) -Task {
-        $webConfigPath = "$($Config.PackageDirectory)/Web.config"
-
-        # Transform the web.config file using the initialization config
-        $transformParams = @{
-            sourceFile = "$($Config.PackageDirectory)/Web.config"
-            transformFile = "$PSScriptRoot/initialization.config"
-            destinationFile = $webConfigPath
-            toolsPath = $Config.ToolsPath
+        $webConfigPath = "$($Config.PackageDirectory)/appsettings.json"
+
+        $InitializationSetting = @{
+            "{ACCOUNT_EMAIL}"    = $Config.AccountEmail
+            "{ACCOUNT_PASSWORD}" = $Config.AccountSecret
+            "{POPULATED_SECRET}" = $Config.PopulatedSecret
+            "{MINIMAL_SECRET}"   = $Config.MinimalSecret
```

Comment:
same thing here as above. these settings need to be added to the existing appsettings. the current structure for the User hashtable that needs to be merged should look something like this but with the values coming from $Config:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/d029844f0d23cb49004e0868a38840c5f92ec7d1/logistics/scripts/modules/settings/settings-management.psm1#L83-L106

Reply:
Changed to read data from config file